wolfcrypt/src/evp.c

```diff
@@ -702,6 +709,16 @@ int wolfSSL_EVP_CipherUpdate(WOLFSSL_EVP_CIPHER_CTX *ctx,
                     return WOLFSSL_SUCCESS;
                 }
             }
+#endif
+#ifdef HAVE_CHACHA
+        case CHACHA20_TYPE:
+        /* CHACHAFIXME */
```

Comment:
Why the FIXME?
No streaming?

Reply:
I am dumb. I added to FIXME to the places where the actual implementation should go, added it and did not remove all FIXMEs. 

force-pushed a rebased version to hide all traces.